README.md

```diff
@@ -4,10 +4,20 @@ SHiELD_driver contains the SHiELD Physics component of the SHiELD atmosphere mod
 
 More information is available on the [GFDL SHiELD page](https://www.gfdl.noaa.gov/shield/).
 
+# History & Attribution
+
+```FV3GFS```, ```GFS_layer```, and ```gsmphys``` contain derivative code (baseline 2017 GFS) plus innovations developed by GFDL during and subsequent to the NGGPS Phase II and III (competition and implementation phases).
+
+```IPD_layer``` was developed by GFDL personnel, with community input, under funding from NWS Office of Science and Technology Integration (mostly superseded within the UFS by the CCPP [framework](https://github.com/NCAR/ccpp-framework) and [physics)](https://github.com/NCAR/ccpp-physics).
```

Comment:
The parenthetical statement about CCPP isn't necessary and gets too much into the "dirty laundry".

Reply:
This is updated in the latest commit